packages/admin/src/pages/scheme/[schemeId]/[exportId]/index.page.tsx

```diff
@@ -0,0 +1,232 @@
+    {
+      name: <>{true && 'Organisation'}</>,
+      width: 'three-quarters',
+    },
+    {
+      name: 'Action',
+      isVisuallyHidden: true,
+    },
+  ] as TheadColumn[];
+
+  const tableRows = unavailableSubmissions.map((submission) => {
+    return {
+      cells: [
+        {
+          content: <>{true && submission.name}</>,
+        },
+        {
+          content: (
+            <div className="govuk-!-text-align-right">
+              <CustomLink
+                href={`/scheme/${schemeId}/${exportId}/view/${submission.submissionId}`}
+                ariaLabel={`View submission "${submission.name}"`}
+              >
+                View
+              </CustomLink>
+            </div>
+          ),
+        },
+      ],
+    };
+  });
+
+  return (
+    <>
+      <Meta title={`Download applications - Manage a grant`} />
+      <div className="govuk-grid-row govuk-!-padding-top-7 govuk-!-margin-0 govuk-!-margin-bottom-6">
+        <div className="govuk-grid-column-full-width">
+          <div className="govuk-!-width-two-thirds">
+            {unavailableSubmissionsTotalCount > 0 && (
+              <DownloadMessage count={unavailableSubmissionsTotalCount} />
+            )}
+
+            <h1 className="govuk-heading-l">{schemeName}</h1>
+
+            <h2 className="govuk-heading-m">Applications submitted</h2>
+
+            <p className="govuk-body">
+              Your grant has{' '}
+              <b>
+                {availableSubmissionsTotalCount}{' '}
+                {availableSubmissionsTotalCount === 1
+                  ? 'application'
+                  : 'applications'}
+              </b>{' '}
+              available to download.
+            </p>
+          </div>
+          <FlexibleQuestionPageLayout
+            fieldErrors={[]}
+            formAction={formAction}
+            csrfToken={csrfToken}
+          >
+            <div className="govuk-button-group">
+              <CustomLink
+                href={`/apply/admin/api/signed-url?key=${encodeURIComponent(
+                  superZipLocation
+                )}`}
+                isButton
+                excludeSubPath
+                disabled={availableSubmissionsTotalCount == 0}
+              >
+                Download all applications
+              </CustomLink>
+
+              <CustomLink
+                href={individualApplicationsHref}
+                isSecondaryButton
+                disabled={availableSubmissionsTotalCount == 0}
+              >
+                View individual applications
+              </CustomLink>
+            </div>
+          </FlexibleQuestionPageLayout>
+
+          {unavailableSubmissions.length > 0 && (
+            <div className="govuk-!-width-two-thirds">
+              <hr className="govuk-section-break govuk-section-break--visible govuk-section-break--m govuk-!-margin-top-2 govuk-!-margin-bottom-7"></hr>
+              <h2 className="govuk-heading-m">Unavailable applications</h2>
```

Comment:
In the figma designs this heading is 'Applications unavailable for download'

Reply:
Will be in next push!